### PR TITLE
bump filestack version

### PR DESCRIPTION
**Pull request by nolman**
I had to use some of the newer filestack features and needed an upgraded filestack version. I don't believe there are any breaking changes in the API.

### CODE REVIEW

**Comment by nolman** [comment]
Added in a `filestack-image` helper /w tests as well so that one can easily generate filestack image links

File: tests/integration/helpers/filestack-image-test.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+test('it handles missing urls', function(assert) {
+  this.set('imageToken', null);
+  this.render(hbs`{{filestack-image imageToken}}`);
+  assert.equal(this.$().text().trim(), '');
+});
+
+test('it handles missing hash arguments', function(assert) {
+  this.set('imageToken', '123ABC');
+  this.render(hbs`{{filestack-image imageToken width='200' height='300' fit=crop}}`);
```
Comment (mminkoff) [inline tests/integration/helpers/filestack-image-test.js]: Unless I misunderstand something, `crop===undefined` here, which is what you're trying to test but isn't obvious.  Perhaps set `const crop = null` first, or just say `fit=null`?

File: addon/helpers/filestack-image.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      urlRoot = 'https://process.filestackapi.com';
+    }
+    return [
+      urlRoot,
+      resizeOptions,
+      token,
+    ].filter((element) => {
+      return element;
+    }).join('/')
+    // return .compact().join('/')
```
Comment (mminkoff) [inline addon/helpers/filestack-image.js]: this comment should be removed, right?

**Comment by nolman** [comment]
One of my co-founders (@scudco) beat me to addressing your feedback @mminkoff. Thanks again for making this library, let us know if you would like anything else taken care of with this PR.

**Comment by mminkoff** [comment]
Version 0.0.8 is available for your installing pleasure, @nolman @scudco

**Comment by mminkoff** [comment]
@nolman @scudco come to think of it, adding a little instruction about the new helper to the README would be nice - want to add something, or even just send me some text and I'll add it?

**Pull request merged**